scapy/asn1/ber.py

```diff
@@ -294,7 +298,7 @@ def safedec(cls, s, context=None):
 
     @classmethod
     def enc(cls, s):
-        if isinstance(s, six.string_types):
+        if isinstance(s, (str, bytes)):
```

Comment:
Why is this needed?

Reply:
That’s for https://github.com/secdev/scapy/issues/1901

It’s a bug from when it was ported to Python 3: it only expects str but should also expect bytes

Reply:
Then we're probably better with this, to accept `unicode` values in Python 2 (as they are the equivalent of `str` in Python 3):
```suggestion
        if isinstance(s, six.string_types + (bytes,)):
```